netbox/extras/graphql/types.py

```diff
@@ -110,5 +111,12 @@ class WebhookType(OrganizationalObjectType):
 
     class Meta:
         model = models.Webhook
-        exclude = ('content_types', )
         filterset_class = filtersets.WebhookFilterSet
+
+
+class EventRuleType(OrganizationalObjectType):
```

Comment:
If we decide to change EventRule to inherit from PrimaryModel, this should be changed to `NetBoxObjectType`.